src/templates/partials/member-card.njk

```diff
@@ -1,20 +1,20 @@
 {% for member in ourTeam.cards %}
-    <li class="our-team__list-item" tabindex="0">
-                <article class="member-card">
-                  <img
+  <li class="our-team__list-item" tabindex="0">
+    <article class="member-card">
+      <img
                     class="member-card__image"
                     src="{{ member.image }}"
                     alt="{ member.alt }}"
                     width="120"
                     height="120"
-                  />
-                  <dl class="member-card__dl">
-                    <dt class="member-card__dt">{{ member.term }}</dt>
-                    <dd class="member-card__dd">{{ member.desc }}</dd>
-                  </dl>
-                    {% set socialClass = "member-card__social" %}
-                    {% set social = ourTeam.cards.socialApps %}
-                    {% include "partials/social.njk" %}   
-                </article>
-    </li>
+/>
+      <dl class="member-card__dl">
+        <dt class="member-card__dt">{{ member.term }}</dt>
+        <dd class="member-card__dd">{{ member.desc }}</dd>
+      </dl>
+      {% set socialClass = "member-card__social" %}
+      {% set social = ourTeam.cards.socialApps %}
+      {% include "partials/social.njk" %}
```

Comment:
Let's look at the example of `social` how we will proceed with styles. Often, if we have a component (also known as a template), then this component should have its own styles. For example for `social.njk` we need to have `social.scss`. In other words, the component has markup and styling. The component expects to receive data, but what kind of data is no longer important, as long as the necessary structure of this data is followed.